avocado/core/sysinfo.py

```diff
@@ -335,15 +335,24 @@ def __init__(self, basedir=None, log_packages=None, profiler=None):
                                            'commands',
                                            key_type='str',
                                            default='')
-        log.info('Commands configured by file: %s', commands_file)
-        self.commands = genio.read_all_lines(commands_file)
+
+        self.commands = []
+        if os.path.isfile(commands_file):
+            log.info('Commands configured by file: %s', commands_file)
+            self.commands = genio.read_all_lines(commands_file)
+        else:
+            log.debug('File %s does not exist.' % commands_file)
```

Comment:
We should stick with the more prevailing style: `log.debug("msg", parameter1, parameter2)`.